ui/components/app/asset-list/asset-list.js

```diff
@@ -69,20 +74,40 @@ const AssetList = ({ onClickAsset }) => {
   const istokenDetectionInactiveOnNonMainnetSupportedNetwork = useSelector(
     getIstokenDetectionInactiveOnNonMainnetSupportedNetwork,
   );
-
+  const tokenList = useSelector(getTokenList);
+  const tokenData = Object.values(tokenList).find(
+    (token) => token.symbol === primaryCurrencyProperties.suffix,
+  );
+  const title = tokenData?.name || primaryCurrencyProperties.suffix;
   return (
     <>
-      <AssetListItem
-        onClick={() => onClickAsset(nativeCurrency)}
-        data-testid="wallet-balance"
-        primary={
-          primaryCurrencyProperties.value ?? secondaryCurrencyProperties.value
-        }
-        tokenSymbol={primaryCurrencyProperties.suffix}
-        secondary={showFiat ? secondaryCurrencyDisplay : undefined}
-        tokenImage={balanceIsLoading ? null : primaryTokenImage}
-        identiconBorder
-      />
+      {process.env.MULTICHAIN ? (
+        <MultichainTokenListItem
+          onClick={() => onClickAsset(nativeCurrency)}
+          title={title}
+          data-testid="wallet-balance"
```

Comment:
You're not spreading props in `MultichainTokenListItem` so I don' think this test id will be attached to the wrapper div. Either spread props, create a test id prop or remove. `MultichainTokenListItem` already has a test id `data-testid="multichain-token-list-item"`
```suggestion
```